.gitignore

```diff
@@ -97,4 +97,9 @@ env.sh
 config.json
 .autoenv.zsh
 
-*~
\ No newline at end of file
+*~
+Pipfile*
```

Comment:
@paddie 
Why do we keep Pipfiles out of our repositories?

Reply:
Ha, good question. Right now, we do it such that every time we execute it, it gets the latest version of the dependency. I guess we could actually include them and just not lock the version at all, but we can talk about it.